package.json

```diff
@@ -16,37 +16,51 @@
   "scripts": {
     "preanalyze-bundles": "webpack --config webpack.browser.config.js --profile --json > stats.json",
     "analyze-bundles": "webpack-bundle-analyzer stats.json .",
+    "postinstall": "node bin/fix-local-config",
     "start": "run-p dev:standalone",
     "dev:standalone": "run-p dev:standalone:*",
     "dev:standalone:js": "NODE_ENV=development webpack-dev-server --config webpack.standalone.config.js",
     "dev:standalone:css": "node-sass src/form.scss | postcss --config postcss.config.json -u autoprefixer -u postcss-custom-properties -o targets/standalone/happychat.css",
+    "dev:npm": "run-p dev:npm:*",
+    "dev:npm:js": "NODE_ENV=development ./node_modules/babel-cli/bin/babel.js --watch src --out-dir targets/npm",
```

Comment:
isn't babel in `node_modules/.bin/`? If so, you can actually do `NODE_ENV=development babel --watch src --out-dir targets/npm` because the scripts will search in `node_module/.bin` path for "binaries".